pkg/diagnose/connectivity/endpoint_info.go

```diff
@@ -0,0 +1,22 @@
+package connectivity
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/forwarder/endpoints"
+	"github.com/DataDog/datadog-agent/pkg/forwarder/transaction"
+)
+
+type EndpointInfo struct {
```

Comment:
`EndpointInfo` doesn't need to be exported.

Reply:
It is actually used in `trace_request.go`. Maybe there is another way to do so that I don't know ?

Reply:
It is in the same package so it should work. 
Note: The fields name can also be not exported.